<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(255, 0, 0), stop:1 rgb(255, 170, 130));
border-radius: 20px;
border: 1px solid red;
color:white;
}
QPushButton:hover
{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(255, 94, 30), stop:1 rgba(255, 208, 176, 200));
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>300</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>100</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_Addmovie">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1000</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>French Script MT</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>add New Movie</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_AddmoviePic">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_MovieList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1000</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>French Script MT</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Movie List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_MoviListPic">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_Users">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1000</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>French Script MT</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Users List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_UserListPic">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>300</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="actionBack"/>
   <addaction name="actionAddMovie"/>
   <addaction name="actionMovieList"/>
   <addaction name="actionUsersList"/>
  </widget>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/Resourse/Resourse/Icon/back.png</normaloff>:/Resourse/Resourse/Icon/back.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Previouse Page&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/Resourse/Resourse/Icon/Exit critical.png</normaloff>:/Resourse/Resourse/Icon/Exit critical.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Exit&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionAddMovie">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/Resourse/Resourse/Icon/icons8-add-file-100.png</normaloff>:/Resourse/Resourse/Icon/icons8-add-file-100.png</iconset>
   </property>
   <property name="text">
    <string>AddMovie</string>
   </property>
  </action>
  <action name="actionMovieList">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/Resourse/Resourse/Icon/icons8-create-100.png</normaloff>:/Resourse/Resourse/Icon/icons8-create-100.png</iconset>
   </property>
   <property name="text">
    <string>MovieList</string>
   </property>
  </action>
  <action name="actionUsersList">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/Resourse/Resourse/Icon/user.png</normaloff>:/Resourse/Resourse/Icon/user.png</iconset>
   </property>
   <property name="text">
    <string>UsersList</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
